build/lib/extensions.js

```diff
@@ -211,6 +211,7 @@ const externalExtensions = [
     'admin-tool-ext-win',
     'agent',
     'arc',
+    'azdata',
```

Comment:
What's this for?

Reply:
I had missed adding this in when I added the azdata extension - Anthony had a PR out which included this as well but I should probably just send out a PR myself. Anyways you can ignore this.

Reply:
nvm, it did get merged in. @t-roup make sure you've synced up to the latest from main